frontend/src/lib/docs.test.ts

```diff
@@ -0,0 +1,130 @@
+import Swagger from '@apidevtools/swagger-parser';
+import { Mock, vi } from 'vitest';
+import { getDocDefinitions, getDocs } from './docs';
+import { request } from './k8s/apiProxy';
+
+vi.mock('@apidevtools/swagger-parser', () => ({
+  default: { dereference: vi.fn() },
```

Comment:
The original issue talks about testing the library, but these tests mock the library, which is the opposite